.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
               run: pnpm types:check
```

Comment:
we can probably rename this one to just `check` as well and we should be good to go

Reply:
done